src/StaticFAQ/FAQArticleNotFound.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, { Component } from 'react';
+import { Heading, Text, Button } from '@kiwicom/orbit-components';
+import { withRouter } from 'react-router-dom';
+import css from 'styled-jsx/css';
+
+const style = css`
+  .notFoundContainer {
+    display: flex;
+    flex: 1;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
+    padding: 0 40px;
+  }
+  .textMargin {
+    margin-top: 10px;
+    margin-bottom: 40px;
+  }
+`;
+
+type Props = {
+  history: {
+    push: string => void,
+  },
+};
+
+class ArticleNotFound extends Component<Props> {
+  goToNoBooking = () => {
```

Comment:
something like `goToFAQ` would be more appropriate, this page might display booking if you are logged in